api/services/mapping/converters.js

```diff
@@ -17,44 +17,41 @@ const FileService = require('../FileService');
 const RiggingService = require('../RiggingService');
 
 const c = {
-  toCave: (source) => {
-    const result = {
-      ...CaveModel,
-    };
-    result.id = source.id;
-    result['@id'] = String(source.id);
+  toCave: (source) => ({
+    ...CaveModel,
+    id: source.id,
+    '@id': String(source.id),
 
-    result.dateInscription = source.date_inscription;
-    result.dateReviewed = source.date_reviewed;
-    result.depth = source.depth;
-    result.isDeleted = source.is_deleted;
-    result.isDiving = source.is_diving;
-    result.latitude = parseFloat(source.latitude);
-    result.length = source.length;
-    result.longitude = parseFloat(source.longitude);
-    result.name = getMainName(source);
-    result.temperature = source.temperature;
+    author: convertIfObject(source.author, c.toCaverShort),
+    reviewer: convertIfObject(source.reviewer, c.toCaverShort),
+    dateInscription: source.dateInscription,
+    dateReviewed: source.dateReviewed,
+    name: getMainName(source),
+    depth: source.depth,
+    length: source.caveLength,
+    temperature: source.temperature,
+    isDeleted: source.isDeleted,
+    isDiving: source.isDiving,
+    latitude: parseFloat(source.latitude),
+    longitude: parseFloat(source.longitude),
 
-    // Convert objects
-    result.author = convertIfObject(source.id_author, c.toCaverShort);
-    result.reviewer = convertIfObject(source.id_reviewer, c.toCaverShort);
-    result.names = toList('names', source, c.toName);
-    result.descriptions = toList('descriptions', source, c.toDescription);
-    result.entrances = toList('entrances', source, c.toEntranceShort);
-    result.documents = toList('documents', source, c.toDocument);
-    result.histories = toList('histories', source, c.toHistory);
-    result.massifs = toList('massifs', source, c.toMassif);
+    names: toList('names', source, c.toName),
+    descriptions: toList('descriptions', source, c.toDescription),
+    entrances: toList('entrances', source, c.toEntranceShort),
+    documents: toList('documents', source, c.toDocument),
+    massifs: toList('massifs', source, c.toMassifShort),
+  }),
 
-    return result;
-  },
   toCaveShort: (source) => ({
```

Comment:
"toCaveShort" sounds a bit odd to me... Maybe I prefer "toSimpleCave", what do you think ?

Reply:
Agree, to remain consistent I also renamed the other methods:
```
toCaveShort -> toSimpleCave
toCaverShort -> toSimpleCaver
toEntranceShort -> toSimpleEntrance
toMassifShort -> toSimpleMassif
```